pyproj/_crs.pyx

```diff
@@ -1660,11 +1673,22 @@ cdef class _CRS(Base):
         warnings.warn("CRS.is_valid is deprecated.")
         return self._type != PJ_TYPE_UNKNOWN
 
+    @property
+    def is_local(self):
```

Comment:
Do we want to use the "local" instead of "engineering"? (since the "engineering" is the proj.4 / ISO terminology?)

"local" is an easier word, but it could also be confused with "local" in the sense of "use a local projected CRS suited for the study area" advice.

Reply:
Sounds good to me.